stixops.py

```diff
@@ -85,11 +85,11 @@ def writeSTIX(self, option):
             name = str (input ("Enter malware name: "))
             family = str (input ("Does the malware have any relations [True/ False]? "))
             description = str (input ("Enter malware description: "))
-            type = str (input ("Enter malware type: "))
```

Comment:
Also individually changed the `type` variables in the respective sections, as `type` is a protected keyword in Python. While this has not been observed to cause issues, it's best to change this.